src/classes/TDTM_DefaultConfig.cls

```diff
@@ -398,7 +403,10 @@ public without sharing class TDTM_DefaultConfig {
         //Flag removed from line 57 of RD_RecurringDonations_TDTM
         npspToCumulusMap.put('npe03__DISABLE_Donations_trigger__c', 'RD_RecurringDonations_Opp_TDTM');
         //Flag removed from line 107 of RD_RecurringDonations_TDTM
-        npspToCumulusMap.put('npe03__DISABLE_RecurringDonations_trigger__c', 'RD_RecurringDonations_TDTM');
+        String rdTriggerHandler = RD2_EnablementService.isEnhancementEnabled
```

Comment:
@force2b , I would rather avoid class and compilation dependency on `RD2_EnablementService`. Is there another entry that can be used in the map for the enhanced recurring donations trigger handler?
@pedrodalcol ^